src/utils/deep-merge.js

```diff
@@ -1,20 +1,30 @@
 // @flow
 
-export default function deepMerge(
-  target?: ?{},
-  ...sources: Array<null | ?{}>
-): {} {
+const MAX_DEPTH_OF_CHECK = 10;
```

Comment:
Where the number `10` comes from?

Reply:
we can replace it with any other likable number. When I did research about redux data-objects in some project, the most optimal size was 6, deeper there was "trash" that sometimes appears in data-object inaccurately mixed in. Other number is 40 as it's a const for stack overflow in JS runtime. I can put here any reasonable number if any